### DISCUSSION TITLE
New to semantic-version

### DISCUSSION THREAD

**Discussion by giorgiog73** [PaulHatch/semantic-version#151] [Q&A] [answered]
Hi, sorry for the silly question. I see it is possible to pass major and minor as parameters in the git branch commit.
So, for example, my commit is git commit -m "Task-xxx:Fixed bugs yyy". How could I include the version that I want to use for the next release? Something like: git commit -m "Task-xxx:Fixed bugs yyy    v2.0.1"  would it work?
Thanks for your help.

**Comment by PaulHatch** [comment, marked as answer]
You can set a tag on the commit to explicitly set the version. This is how the version is set during [typical usage](https://github.com/PaulHatch/semantic-version/blob/master/guide.md#tag-versioning).

**Reply by giorgiog73** [reply]
Thank you :)
